test/jdk/tools/pack200/PackTestZip64.java

```diff
@@ -85,43 +85,53 @@ static void testPacking() throws IOException {
         cmdsList.add(unpackedFile.getName());
         Utils.runExec(cmdsList);
 
-        // Compare files binary
-        compareTwoFiles(testFile, unpackedFile);
+        if (!testFile.exists()) {
+            throw new IOException("File " + testFile.getName() + " does not exist!");
+        }
+
+        if(!unpackedFile.exists()) {
+            throw new IOException("File " + unpackedFile.getName() + " does not exist!");
+        }
+        // Compare two Jar file
```

Comment:
typo: file -> files

Reply:
Fixed in https://github.com/ibmruntimes/openj9-openjdk-jdk11/compare/3232726537abaceda9f08afd1521668f3f8704c4..18b42c0619ab09f595149f0dacc6c70521482ac8